operate/client/src/App/Processes/ListView/DiagramPanel/DiagramHeader/index.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {observer} from 'mobx-react';
+import isNil from 'lodash/isNil';
+import {CopiableProcessID} from 'App/Processes/CopiableProcessID';
+import {ProcessOperations} from '../../ProcessOperations';
+import {Restricted} from 'modules/components/Restricted';
+import {processesStore} from 'modules/stores/processes/processes.list';
+import {PanelHeader, Dd, Dl, Dt} from './styled';
+
+type ProcessDetails = {
+  bpmnProcessId?: string;
+  processName: string;
+  version?: string;
+  versionTag?: string | null;
+};
+
+type DiagramHeaderProps = {
+  processDetails: ProcessDetails;
+  processDefinitionId?: string;
+  tenant?: string;
+  isVersionSelected: boolean;
+  panelHeaderRef?: React.RefObject<HTMLDivElement>;
+};
+
+const DiagramHeader: React.FC<DiagramHeaderProps> = observer(
+  ({
+    processDetails,
+    processDefinitionId,
+    tenant,
+    isVersionSelected,
+    panelHeaderRef,
+  }) => {
+    const {
+      processName,
+      bpmnProcessId,
+      version,
+      // TODO: revert
```

Comment:
Not sure what this is for, but just a reminder to not forget it.

Reply:
It was for design review purposes. I reverted it.